Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -328,6 +337,8 @@ u32 Renderer::AccessEFB(EFBAccessType type, u32 x, u32 y, u32 poke_data)
 
 void Renderer::PokeEFB(EFBAccessType type, const EfbPokeData* points, size_t num_points)
 {
+  LWDProfiler::ScopedMarker marker("RenderBase/Poke EFB");
```

Comment:
Poke EFB is rarely used. but it can be very hot in games that do use it. 
Not a great place for a marker